client/app/queue/AddCavcRemandView.jsx

```diff
@@ -180,14 +186,14 @@ const AddCavcRemandView = (props) => {
   const submit = () => {
     const payload = {
       data: {
-        judgement_date: mandateAvailable() ? judgementDate : null,
-        mandate_date: mandateAvailable() ? mandateDate : null,
+        judgement_date: (remandType() && mdrSubtype()) || !mandateAvailable() ? null : judgementDate,
+        mandate_date: (remandType() && mdrSubtype()) || !mandateAvailable() ? null : mandateDate,
```

Comment:
I think you mean this:
```suggestion
        judgement_date: ((remandType() && mdrSubtype()) || !mandateAvailable()) ? null : judgementDate,
        mandate_date: ((remandType() && mdrSubtype()) || !mandateAvailable()) ? null : mandateDate,
```
A test would have caught this bug.